bin/MadGraph5_aMCatNLO/cards/production/13TeV/TTToHcToWA_AToMuMu/TTToHcToWA_AToMuMu_MHc160_MA120/TTToHcToWA_AToMuMu_MHc160_MA120_customizecards.dat

```diff
@@ -0,0 +1,10 @@
+# Define Higgs mass
+# 25:h, 35:H, 36: A, 37: Hc
+set param_card mass 25 125.1
```

Comment:
Is this fine tuning or a typo? Not that it makes a difference compared to 125 GeV. Just wondering. In a paper it might be easier to describe the MC sample saying "and a higgs width of 125 GeV".

Reply:
I just set the higgs mass and width as the value in PDG, as assuming the light CP even higgs as the SM like higgs.